info.limpet/src/info/limpet/data/store/InMemoryStore.java

```diff
@@ -83,12 +84,20 @@ public boolean remove(Object o)
 			return res;
 		}
 
+		@UIProperty(name="Name", category="Label")
```

Comment:
In my opinion, it would be worth declaring some category labels as public static, in some top level file.  Maybe in UIProperty.